scripts/superbuild/build_ospray.cmake

```diff
@@ -32,8 +32,10 @@ ExternalProject_Add(${COMPONENT_NAME}
     -DOSPRAY_INSTALL_DEPENDENCIES=${INSTALL_DEPENDENCIES}
     -DOSPRAY_STRICT_BUILD=${BUILD_OSPRAY_CI_EXTRAS}
     -DOSPRAY_WARN_AS_ERRORS=${BUILD_OSPRAY_CI_EXTRAS}
+    -DOSPRAY_ENABLE_APPS=${OSPRAY_ENABLE_APPS}
```

Comment:
I agree, we should add a superbuild option `BUILD_OSPRAY_APPS`, and let also `OSPRAY_MPI_BUILD_TUTORIALS` depend on it. I'll give it this try.